hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteFunction.java

```diff
@@ -100,8 +100,8 @@
  * @param <I> Type of the input record
  * @see StreamWriteOperatorCoordinator
  */
-public class StreamWriteFunction<K, I, O>
-    extends KeyedProcessFunction<K, I, O>
+public class StreamWriteFunction<I, O>
+    extends ProcessFunction<I, O>
```

Comment:
Looks like even we need the keyBy, we can still use the `ProcessFunction` here instead of `KeyedProcessFunction` right

Reply:
Yes, because the key we doesn't use in StreamWriteFunction.